editoast/src/main.rs

```diff
@@ -184,7 +191,50 @@ async fn run() -> Result<(), Box<dyn Error + Send + Sync>> {
             }
             InfraCommands::ImportRailjson(args) => import_railjson(args, create_db_pool()?).await,
         },
+        Commands::Trains(subcommand) => match subcommand {
+            TrainsCommands::Import(args) => trains_import(args, create_db_pool()?).await,
+        },
+    }
+}
+
+async fn trains_import(
+    args: ImportTrainArgs,
+    db_pool: Data<DbPool>,
+) -> Result<(), Box<dyn Error + Send + Sync>> {
+    let conn = &mut db_pool.get().await?;
+    let timetable = match args.timetable {
+        Some(timetable) => match Timetable::retrieve(conn, timetable.parse::<i64>()?).await? {
+            Some(timetable) => timetable,
+            None => {
+                let error = CliError::new(1, format!("❌ Timetable not found, id: {timetable}"));
+                return Err(Box::new(error));
+            }
+        },
+        None => {
+            let changeset = Timetable::changeset();
+            changeset.create(conn).await?
+        }
+    };
+
+    let train_file = File::open(args.path)?;
+    let train_schedules: Vec<TrainScheduleBase> =
+        serde_json::from_reader(BufReader::new(train_file))?;
+    let ts_length = train_schedules.len();
+
+    for train_schedule in train_schedules {
+        let form = TrainScheduleForm {
+            timetable_id: timetable.id,
+            train_schedule,
+        };
+        let changeset: TrainScheduleChangeset = form.into();
+        changeset.create(conn).await?;
```

Comment:
We should first create a `Vec<TrainScheduleChangeset>` and then use `TrainSchedule::create_batch(...)`. You can use the POST endpoint as an example.